installer/driverInstall.js

```diff
@@ -36,17 +36,31 @@ var install_node_ibm_db = function(file_url) {
 
             //Windows node binary names should update here.
             var ODBC_BINDINGS = 'build\/Release\/odbc_bindings.node';
-            var ODBC_BINDINGS_V10 = 'build\/Release\/odbc_bindings.node.0.10.36';
             var ODBC_BINDINGS_V12 = 'build\/Release\/odbc_bindings.node.0.12.7';
             var ODBC_BINDINGS_V4 = 'build\/Release\/odbc_bindings.node.4.6.1';
             var ODBC_BINDINGS_V6 = 'build\/Release\/odbc_bindings.node.6.9.1';
 
+            //ERROR: NodeJs version < 0.12.x Support for node-ibm_db Windows has been discontinued.
+            if(Number(process.version.match(/^v(\d+\.\d+)/)[1]) < 0.12){
```

Comment:
There are some reasons having to do with package installation order in npm2 to not use your dependencies in install scripts, though, so perhaps the best thing would be just to implement a `function semverLessThan() ` and use it to do comparisons. It should break the major and minor apart and compare them seperately to not have the "versions miscompared as floats" problem.